crates/stdarch-gen-loongarch/README.md

```diff
@@ -0,0 +1,35 @@
+# LoongArch LSX/LASX intrinsic code generator
+
+A small tool that allows to quickly generate intrinsics for the LoongArch LSX/LASX architectures.
+
+The specification for the intrinsics can be found in `lsx.spec` or `lasx.spec`.
+
+To run and re-generate the code run the following from the root of the `stdarch` crate.
+
+LSX:
+```
+# Generate bindings
+OUT_DIR=`pwd`/crates/stdarch-gen-loongarch cargo run -p stdarch-gen-loongarch -- crates/stdarch-gen-loongarch/lsxintrin.h
+OUT_DIR=`pwd`/crates/core_arch cargo run -p stdarch-gen-loongarch -- crates/stdarch-gen-loongarch/lsx.spec
+
+# Generate tests
+OUT_DIR=`pwd`/crates/stdarch-gen-loongarch cargo run -p stdarch-gen-loongarch -- crates/stdarch-gen-loongarch/lsx.spec test
+# On LoongArch
+gcc -o lsx crates/stdarch-gen-loongarch/lsx.c -mlasx
```

Comment:
Can you change these instructions to use qemu-loongarch64 and a cross-compiler (loongarch64-linux-gnu-gcc)? This makes it easier for people to run without access to loongarch hardware.

Reply:
I tested on [this git commit](https://github.com/qemu/qemu/commit/dccbaf0cc0f1744ffd7562a3dc60e4fc99fd9d44) of QEMU using qemu-loongarch64 and a cross-compiler (loongarch64-unknown-linux-gnu-gcc), and I was able to generate the correct test cases. Thanks.